atomic_reactor/schemas/container.json

```diff
@@ -82,6 +82,9 @@
       },
       "additionalProperties": true
     },
+    "inherit_from_parent": {
```

Comment:
Should this be part of "compose"? It feels like it's confined to that area.

Reply:
problem is that it can also be present even jut to inherit yum repo, when compose isn't present

Reply:
so I've made it not dependent to compose

Reply:
Sure, but none of the other properties inside 'compose' are required. It could literally be:
```
compose:
  inherit_from_parent: true
```

It just looks like a namespacing issue to me.

Reply:
right it should be fine in compose anyway, moving it there